docs/beyond_basics/101-181-metalad.rst

```diff
@@ -4,10 +4,276 @@ Metadata-Management with MetaLad
+	But there are also some internal checks that might be surprising.
+	If you want to add the metadata above to your own dataset, you should make sure to adjust the ``dataset_id`` to the ID of your own dataset, found via the command ``datalad configuration get datalad.dataset.id`` - otherwise you'll see an error [#f4]_.
+	And in case you'd supply the ``extraction_time`` as "this morning at 8AM" instead of a time stamp, the command will be unhappy as well.
+	In case an error occurs, make sure to read the error message, and turn the the commands' ``--help`` for insights about requirements you might have missed.
+
+After the metadata has been added, you can view it via the command :command:`meta-dump`.
+The simplest form of this command is ``meta-dump -r``, which will show all metadata that is stored in dataset in the current directory.
+To get more specific metadata records, you can give a dataset-file-path-pattern to ``meta-dump``, much like an argument to ``ls``, that identifies :term:`dataset ID`, version and a file within the dataset.
+The two parts are separated by ``:``. The following line would just dump all metadata for ``zen.png``.
+
+.. code-block:: bash
+
+    $ datalad meta-dump -d cozy-screensavers .:zen.png
+    TODO: how would the output of this look like?
+
+.. find-out-more:: More complex metadata-dumps
+
+   TODO: add complex Dataset-file-path-pattern examples, e.g., with UUIDs, versions, etc
+
+Using existing extractors to add metadata
+"""""""""""""""""""""""""""""""""""""""""
+
+TODO: This needs and example
+
+
+Creating your own extractor
+"""""""""""""""""""""""""""
+
+TODO: This also needs an example
+
+
+Distributing and Getting Metadata
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once metadata has been added to a dataset, it can be distributed and retrieved.
+Instead of creating and adding metadata yourself, you could download fitting pre-existing metadata.
+Similarly, instead of repeating a ``meta-add`` process for one and the same files across hierarchies of datasets, metadata added into one dataset can be exported into other datasets.
+Regardless of whether it is a distribution or a retrieval process, though, an export with MetaLad will only concern the *metadata*, and never the primary data.
+
+Download Metadata from a remote repository
+""""""""""""""""""""""""""""""""""""""""""
+
+Let's start by creating a place where someone else's metadata could live.
+
+.. code-block:: bash
+
+    $ datalad create metadata-assimilation
+    $ cd metadata-assimilation
+
+Because MetaLad stores metadata in :term:`Git`'s object store, we use Git to directly fetch metadata from a remote repository, such as this demo on :term:`GitHub`: ``https://github.com/christian-monch/metadata-test.git``.
+Because metadata added by MetaLad is not transported automatically but needs to be specifically requested, the command to retrieve it looks unfamiliar to non-Git-users: It identifies the precise location of :term:`ref` that contains the metadata.
```

Comment:
```suggestion
Because metadata added by MetaLad is not transported automatically but needs to be specifically requested, the command to retrieve it looks unfamiliar to non-Git-users: It identifies the precise location of the :term:`ref` that contains the metadata.
```